source/_integrations/imap.markdown

```diff
@@ -180,7 +182,61 @@ Available actions are:
 - `seen`: Mark the message as seen.
 - `move`: Move the message to a `target_folder` and optionally mark the message `seen`.
 - `delete`: Delete the message.
-- `fetch`: Fetch the content of a message. Returns a dictionary containing `"text"`, `"subject"`, `"sender"` and `"uid""`. This allows to fetch and process the complete message text, not limited by size.
+- `fetch`: Fetch the content of a message.
```

Comment:
The service data is not documented, as a user I have no idea how to call the services. Can we add that in a separate PR?